specification/redis/resource-manager/Microsoft.Cache/stable/2020-12-01/redis.json

```diff
@@ -0,0 +1,2696 @@
+    },
+    "RedisUpdateProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "The SKU of the Redis cache to deploy."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/RedisCommonProperties"
+        }
+      ],
+      "description": "Patchable properties of the redis cache.",
+      "x-ms-external": true
+    },
+    "RedisProperties": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Redis instance provisioning status.",
+          "enum": [
+            "Creating",
+            "Deleting",
+            "Disabled",
+            "Failed",
+            "Linking",
+            "Provisioning",
+            "RecoveringScaleFailure",
+            "Scaling",
+            "Succeeded",
+            "Unlinking",
+            "Unprovisioning",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "hostName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Redis host name."
+        },
+        "port": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Redis non-SSL port."
+        },
+        "sslPort": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Redis SSL port."
+        },
+        "accessKeys": {
+          "readOnly": true,
+          "$ref": "#/definitions/RedisAccessKeys",
+          "description": "The keys of the Redis cache - not set if this object is not the response to Create or Update redis cache"
+        },
+        "linkedServers": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RedisLinkedServer"
+          },
+          "description": "List of the linked servers associated with the cache"
+        },
+        "instances": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RedisInstanceDetails"
+          },
+          "description": "List of the Redis instances associated with the cache"
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v1/privatelinks.json#/definitions/PrivateEndpointConnection"
+          },
+          "description": "List of private endpoint connection associated with the specified redis cache"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/RedisCreateProperties"
+        }
+      ],
+      "description": "Properties of the redis cache.",
+      "x-ms-external": true
+    },
+    "Resource": {
```

Comment:
@zhenglaizhang I tried that here. It says breaking change python. Is it a good breaking change or a bad one?

https://github.com/Azure/azure-rest-api-specs/pull/14015

Reply:
@zhenglaizhang Looks like the tooling probably just broke because its a PR against the dev branch. I don't seem to have permission to push it directly, so would that be the best I can do? Or I could change the source branch for this PR maybe.

Reply:
@zhenglaizhang Actually it doesn't seem possible to change source branch, only target. Can you please merge the fixup PR #14015 into the dev branch to see if that makes everything better?